src/provider/provider_tracking.c

```diff
@@ -119,6 +121,159 @@ static umf_result_t trackingAlloc(void *hProvider, size_t size,
     return ret;
 }
 
+static umf_result_t trackingAllocationSplit(void *provider, void *ptr,
+                                            size_t totalSize,
+                                            size_t firstSize) {
+    umf_result_t ret = UMF_RESULT_ERROR_UNKNOWN;
+    umf_tracking_memory_provider_t *p =
```

Comment:
could we stay with the full name? During the review I had to scroll up to see what is "p". If you want to keep the code short it would be better to add an alias for "tracker = provider->hTracker"